### PR TITLE
[Junos]: fqcn_doc_updates

### PR DESCRIPTION
**Pull request by rohitthakur2590**
Signed-off-by: Rohit Thakur <rohitthakur2590@outlook.com>
##### ISSUE TYPE

 
- Feature Pull Request
 

##### COMPONENT NAME

junos.py
junos_acl_interfaces.py
junos_acls.py
junos_banner.py
junos_command.py
junos_config.py
junos_facts.py
junos_interface.py
junos_interfaces.py
junos_l2_interface.py
junos_l2_interfaces.py
junos_l3_interface.py
junos_l3_interfaces.py
junos_lacp.py
junos_lacp_interfaces.py
junos_lag_interfaces.py
junos_linkagg.py
junos_lldp.py
junos_lldp_global.py
junos_lldp_interface.py
junos_lldp_interfaces.py
junos_logging.py
junos_netconf.py
junos_package.py
junos_ping.py
junos_rpc.py
junos_scp.py
junos_static_route.py
junos_static_routes.py
junos_system.py
junos_user.py
junos_vlan.py
junos_vlans.py
junos_vrf.py
junos.py

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: plugins/modules/junos_interface.py [hunk 1]
```diff
@@ -10,18 +10,18 @@
 
 
 DOCUMENTATION = """
----
 module: junos_interface
-version_added: "1.0.0"
 author: Ganesh Nalawade (@ganeshrn)
-short_description: (deprecated) Manage Interface on Juniper JUNOS network devices
+short_description: (deprecated, removed after 2022-06-01) (deprecated, removed after
```
Comment (ganeshrn) [inline plugins/modules/junos_interface.py:15]: The comment is duplicated here. I have added a fix in runtime.yml to fix it. https://github.com/ansible-network/collection_prep/pull/14

Comment (rohitthakur2590) [reply plugins/modules/junos_interface.py]: @ganeshrn  thanks, I have updated the PR  with your changes.

File: plugins/modules/junos_l2_interface.py [hunk 2]
```diff
@@ -90,6 +88,11 @@
 extends_documentation_fragment:
 - junipernetworks.junos.junos
 
+
+
+
+
+
```
Comment (ganeshrn) [inline plugins/modules/junos_l2_interface.py:95]: Is this space required?

Comment (rohitthakur2590) [reply plugins/modules/junos_l2_interface.py:95]: @ganeshrn definitely not required.  thanks for finding this, I'll correct this.

File: plugins/modules/junos_l3_interfaces.py [hunk 3]
```diff
@@ -133,47 +133,8 @@
...
-#             address 10.200.16.10/24;
-#         }
-#     }
-# }
-# ge-0/0/2 {
-#     description "non configured interface";
-#     unit 0;
-# }
-
-- name: Merge provided configuration with device configuration (default operation is merge)
```
Comment (ganeshrn) [inline plugins/modules/junos_l3_interfaces.py:176]: Why are these lines deleted?

File: plugins/modules/junos_lldp_interface.py [hunk 4]
```diff
@@ -54,6 +54,8 @@
 extends_documentation_fragment:
 - junipernetworks.junos.junos
 
+
```
Comment (ganeshrn) [inline plugins/modules/junos_lldp_interface.py:57]: Is the space required?

File: plugins/modules/junos_vlan.py [hunk 5]
```diff
@@ -70,6 +70,8 @@
 extends_documentation_fragment:
 - junipernetworks.junos.junos
 
+
+
```
Comment (ganeshrn) [inline plugins/modules/junos_vlan.py:74]: Same as above

File: plugins/modules/junos_l3_interfaces.py (hunk 3 above)
Comment (rohitthakur2590) [reply plugins/modules/junos_l3_interfaces.py:176]: It is a mistake, will update this. thanks

File: plugins/modules/junos_lldp_interface.py (hunk 4 above)
Comment (rohitthakur2590) [reply plugins/modules/junos_lldp_interface.py:57]: Looks like something up with the script, anyway I am going to remove similar extra lines from all these modules.

**Comment by rohitthakur2590** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck
